packages/pluggableWidgets/datagrid-web/src/components/Table.tsx

```diff
@@ -125,18 +136,46 @@ export function Table<T>(props: TableProps<T>): ReactElement {
                 },
                 Cell: ({ cell, value }) =>
```

Comment:
:owl:  Why we put resizer only on the top row? Why not on header or all rows?

Reply:
Because header is in another scope due to infinite body so it is another CSS Grid

Reply:
MM, so i cannot resize by dragging border between headers? But it is probably the most used way of doing it

Reply:
Other question: if resizing is broken (according to description), maybe better to completely remove it from this PR?

Reply:
Thats why WT-2790 is open 😄

Reply:
Yes, but it makes it very hard to reason about these code changes. So I would say - if resizing doesn't work, let's kill it completely. And resurrect in next story.

Reply:
Agreed, gonna remove

Reply:
But, this is not the resizer function, this is the content size detector for when "Auto fit content" is enabled.